bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/Runtime.java

```diff
@@ -152,4 +179,13 @@ public abstract Object invokeMethodAsync(BObject object, String methodName, Stri
     public abstract void deregisterListener(BObject listener);
 
     public abstract void registerStopHandler(BFunctionPointer<?, ?> stopHandler);
+
+    /**
+     * Invoke a Ballerina function pointer asynchronously.
+     *
+     * @param functionName  Name of the function which needs to be invoked.
+     * @param args          Arguments of the Ballerina function.
+     * @param callback      Callback which will get notified once the function execution is done.
+     */
+    public abstract void invokeMethodAsync(String functionName, Object[] args, Callback callback);
```

Comment:
```suggestion
    public abstract void invokeMethodAsync(String functionName, Callback callback, Object... args);
```

Reply:
Shall we change the method as below to comply with other invokeMethodAsync methods

Reply:
Addressed with https://github.com/ballerina-platform/ballerina-lang/pull/42363/commits/85939a40a21405befa32e18dcf654fa68f849c49